From e5952a0cdfa8b1b56a5823574835f1f771f14ae0 Mon Sep 17 00:00:00 2001
From: Marcelo Ricardo Leitner <marcelo.leitner@gmail.com>
Date: Fri, 24 Aug 2018 09:53:00 -0300
Subject: [PATCH] build: 0b0dce7a36fb actually belongs to v4.19

Typo or not, this commit actually belongs to v4.19 and made me wonder why
on v4.18 it didn't find this feature.

Fixes: 817f0bfa248f ("build: add two defines for Peer Address Parameters extensions on sctp_paddrparams")
Signed-off-by: Marcelo Ricardo Leitner <marcelo.leitner@gmail.com>
Reviewed-by: Xin Long <lucien.xin@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index f55775a..5de5c76 100644
--- a/configure.ac
+++ b/configure.ac
@@ -101,7 +101,7 @@ LKSCTP_CHECK_TYPE([struct sctp_prinfo], [HAVE_SCTP_SENDV])
 # added on v4.16, 30f6ebf65bc4
 LKSCTP_CHECK_DECL([SCTP_AUTH_NO_AUTH], [HAVE_SCTP_AUTH_NO_AUTH])
 
-# New members to sctp_paddrparams, added on v4.18, 0b0dce7a36fb
+# New members to sctp_paddrparams, added on v4.19, 0b0dce7a36fb
 LKSCTP_CHECK_MEMBER([struct sctp_paddrparams.spp_ipv6_flowlabel],
 		    [HAVE_SCTP_SPP_IPV6_FLOWLABEL])
 LKSCTP_CHECK_MEMBER([struct sctp_paddrparams.spp_dscp],
-- 
2.7.4

